Can't deploy to a roboRIO 2

I have a RIO 1 and RIO 2 set up.  They have the same:
- Image (2022.2)
- Team number (1718)

I am using the same USB A-to-B cable to connect to each RIO (one RIO at a time).

I can reproduce the following.
1. Create a blank WPILib robot project (C++ or Java, reproduces with both)
2. Connect USB to the roboRIO 1
3. Driver Station shows comms
4. Deploy the robot project [Project deploys succesfully]
5. Disconnect the USB cable
6. Driver Station shows no comms
7. Connect USB to the roboRIO 2
8. Driver Station shows comms again
9. Deploy the robot project [deploy fails, cannot find the roboRIO 2]

I can switch back and forth between connecting to the RIO 1 and RIO 2, and reliably the deploy is successful for the RIO 1 and unsuccessful for the RIO 2.

Error Message:
```
Driver Station reported IP: 172.22.11.2

> Task :discoverroborio
Discovering Target roborio

> Task :discoverroborio FAILED
Missing Target!
=============================================
Are you connected to the robot, and is it on?
=============================================
GradleRIO detected this build failed due to not being able to find "roborio"!
Scroll up in this error log for more information.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':discoverroborio'.
> A failure occurred while executing edu.wpi.first.deployutils.deploy.target.discovery.TargetDiscoveryWorker
   > Target roborio could not be found at any location! See above for more details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
4 actionable tasks: 2 executed, 2 up-to-date
The terminal process "cmd.exe /d /c gradlew deploy  -PteamNumber=1718 --offline  -Dorg.gradle.java.home="C:\Users\Public\wpilib\2022\jdk"" terminated with exit code: 1.

Terminal will be reused by tasks, press any key to close it.
```

--- a/src/main/java/edu/wpi/first/gradlerio/deploy/roborio/RoboRIO.java
+++ b/src/main/java/edu/wpi/first/gradlerio/deploy/roborio/RoboRIO.java
@@ -141,7 +141,7 @@ public class RoboRIO extends WPIRemoteTarget {
 
     private void readAndVerifyImage(DeployContext context) {
         final String imageFile = "/etc/natinst/share/scs_imagemetadata.ini";
-        final Pattern pattern = Pattern.compile("^IMAGEVERSION\\s*=\\s*\\\"FRC_roboRIO(?:2)?_(\\d{4}_v\\d+(?:\\.\\d+)?)\\\"");
+        final Pattern pattern = Pattern.compile("^IMAGEVERSION\\s*=\\s*\\\"FRC_roboRIO2?_(\\d{4}_v\\d+(?:\\.\\d+)?)\\\"");
 
         String content = context.execute("cat " + imageFile).getResult();
         log.info("Received Image File: ");

